alembic/versions/2024_01_25_486e8c875e0b_add_upload_jobs.py

```diff
@@ -0,0 +1,34 @@
+"""Add upload jobs
+
+Revision ID: 486e8c875e0b
+Revises: 6c3da78d1af0
+Create Date: 2024-01-25 10:15:59.984529
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "486e8c875e0b"
+down_revision = "6c3da78d1af0"
```

Comment:
Just a heads up. Be sure to double check the revision/down_revision id's I have seen a lot of alembic prs the last days, just to avoid getting merge conflicts and id's being out of sync!